libethash-cl/CLMiner.cpp

```diff
@@ -29,6 +29,7 @@ struct CLChannel: public LogChannel
 };
 #define cllog clog(CLChannel)
 #define ETHCL_LOG(_contents) cllog << _contents
+#define ETHCL_ERR(_msg, _what, _err) cwarn << _msg":" << _what << "," << CLMiner::strClError(_err) << "(" << _err << ")" 
```

Comment:
Do not use macros. I'm ok with repeating this pattern 3 times.

Reply:
@chfast what about some (inline?) function like ``ethcl_err(const char * msg, const char * what, cl_int err)``?

Reply:
I would prefer to keep the `<<` syntax`. At least keep `cwarn << yourClErrorHelper(...)`.